fission-core/src/capabilities/did.rs

```diff
@@ -0,0 +1,42 @@
+//! DID resource
+
+use anyhow::{anyhow, Result};
+use ucan::capability::Scope;
+use url::Url;
+
+//////////////
```

Comment:
Separations are fine. But, it'd be nicer to have comments on all `pub` code, as that will show up in rust docs.

Reply:
Good point, will change.